cmd/opm/registry/add.go

```diff
@@ -27,7 +27,6 @@ func newRegistryAddCmd() *cobra.Command {
 	rootCmd.Flags().StringP("database", "d", "bundles.db", "relative path to database file")
 	rootCmd.Flags().StringSliceP("bundle-images", "b", []string{}, "comma separated list of links to bundle image")
 	rootCmd.Flags().Bool("permissive", false, "allow registry load errors")
-	rootCmd.Flags().StringP("container-tool", "c", "podman", "tool to interact with container images (save, build, etc.). One of: [docker, podman]")
```

Comment:
we should consider making this a hidden flag in case anyone was relying on it existing (instead of throwing an error about not knowing the flag).

If we're confident no one is relying on it yet, we can remove.

Reply:
@benluddy and I were discussing this in [an earlier review](). Within Red Hat, I know of a handful unreleased automation making use of the default value (not specifying `--container-tool`). However, I do think it's safer to use a hidden flag. Can we meet in the middle with a hidden flag that logs a warning when specified and continues with the daemonless implementation?

Reply:
I made it a deprecated flag, which is both hidden and gives a warning when used.